lineage_test.go

```diff
@@ -33,7 +33,8 @@ func TestBindLineage(t *testing.T) {
 		}
 
 		if err != nil {
-			tc.Fatalf("error binding lineage: %+v", err)
+			tc.ValidateErrorOrFail(err)
```

Comment:
can get rid of this by just having a test where failure is the expected output - which is what should just happen by moving the fixtures over to `testdata/invalidlineage`